Documentation/upgrades/upgrade_3_0.md

```diff
@@ -124,5 +124,6 @@ $ ETCDCTL_API=3 etcdctl endpoint health
 
 - etcd &lt; v3.1 does not work properly if built with Go &gt; v1.7. See [Issue 6951](https://github.com/coreos/etcd/issues/6951) for additional information.
 - If an error such as `transport: http2Client.notifyError got notified that the client transport was broken unexpected EOF.` shows up in the etcd server logs, be sure etcd is a pre-built release or built with (etcd v3.1+ &amp; go v1.7+) or (etcd &lt;v3.1 &amp; go v1.6.x).
```

Comment:
1.7+ is 1.7 and above (e.g., 1.8). 1.7.x is all versions of 1.7 (e.g., 1.7.1 but not 1.8.0)